src/Aspire.Hosting.Azure/AzureCosmosDBCloudApplicationBuilderExtensions.cs

```diff
@@ -18,7 +18,7 @@ public static class AzureCosmosDBCloudApplicationBuilderExtensions
     /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
     /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
     /// <param name="connectionString">The connection string.</param>
-    /// <returns>A reference to the <see cref="IResourceBuilder{AzureCosmosDatabaseResource}"/>.</returns>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
```

Comment:
can you explain these changes? it's not returning an unconstrained IResourceBuilder, etc.

Reply:
XML docs don't handle real types (it generates IResourceBuilder`1). VS IntelliSense doesn't care either. You can type anything you want. What I saw was discrepancies due to copy pastes that were cloning the wrong types actually. It didn't matter in the end but with nothing will be semantically wrong.

For reference how I learned that: https://github.com/dotnet/aspire/pull/1480#discussion_r1434631705

Reply:
got it